test/getAssetFromKV.js

```diff
@@ -2,29 +2,8 @@ import test from 'ava'
 import { mockGlobal, getEvent } from '../src/mocks'
 import { getAssetFromKV, mapRequestToAsset } from '../src/index'
 
-test('mapRequestToAsset() correctly changes /about -> /about/index.html', async t => {
-  mockGlobal()
-  let path = '/about'
-  let request = new Request(`https://foo.com${path}`)
-  let newRequest = mapRequestToAsset(request)
-  t.is(newRequest.url, request.url + '/index.html')
-})
-
-test('mapRequestToAsset() correctly changes /about/ -> /about/index.html', async t => {
-  let path = '/about/'
-  let request = new Request(`https://foo.com${path}`)
-  let newRequest = mapRequestToAsset(request)
-  t.is(newRequest.url, request.url + 'index.html')
-})
-
-test('mapRequestToAsset() correctly changes /about.me/ -> /about.me/index.html', async t => {
-  let path = '/about.me/'
-  let request = new Request(`https://foo.com${path}`)
-  let newRequest = mapRequestToAsset(request)
-  t.is(newRequest.url, request.url + 'index.html')
-})
-
 test('getAssetFromKV return correct val from KV and default caching', async t => {
+  mockGlobal()
```

Comment:
why not setting global?

Reply:
what do you mean?

Reply:
Curious why this line needed changing

Reply:
i'm not sure? it is using `Request`, which is part of this global thing, and was apparently not being called before?

Reply:
ooooo mockGlobal is called once per file it looks like.